public/friendlycode/js/fc/ui/editor-panes.js

```diff
@@ -2,65 +2,27 @@ define(function(require) {
   var $ = require("jquery"),
       Slowparse = require("slowparse/slowparse"),
       ParsingCodeMirror = require("fc/ui/parsing-codemirror"),
-      Help = require("fc/help"),
-      LivePreview = require("fc/ui/live-preview"),
-      ErrorHelp = require("fc/ui/error-help"),
-      ContextSensitiveHelp = require("fc/ui/context-sensitive-help"),
-      Relocator = require("fc/ui/relocator"),
-      HelpMsgTemplate = require("template!help-msg"),
-      ErrorMsgTemplate = require("template!error-msg");
+      Help = require("fc/help");
 
   require('slowparse-errors');
-  require("codemirror/html");
 
   return function EditorPanes(options) {
     var self = {},
         div = options.container,
-        initialValue = options.value || "",
+        initialValue = options.value || "(none)",
         allowJS = options.allowJS || false,
-        sourceCode = $('<div class="source-code"></div>').attr('id','webmaker-source-code-pane').appendTo(div),
-        previewArea = $('<div class="preview-holder"></div>').attr('id','webmaker-preview-holder-pane').appendTo(div),
-        helpArea = $('<div class="help hidden"></div>').appendTo(div),
-        errorArea =  $('<div class="error hidden"></div>').appendTo(div);
+        editor = $('<div class="source-code"></div>').attr('id','webmaker-source-code-pane').appendTo(div);
 
-    var codeMirror = self.codeMirror = ParsingCodeMirror(sourceCode[0], {
-      mode: "text/html",
-      theme: "jsbin",
-      tabMode: "indent",
-      lineWrapping: true,
-      lineNumbers: true,
-      gutters: [
-        "CodeMirror-linenumbers",
-        "gutter-markers"
-      ],
-      value: initialValue,
+    // This is not an actual codemirror instance. See: bramble-proxy.js
+    var codeMirror = self.codeMirror = ParsingCodeMirror(editor, {
       parse: function(html) {
         return Slowparse.HTML(document, html, {
```

Comment:
I've been trying to avoid doing too much of an audit of the friendlycode source in an effort to get to mvp. Is it worth going through it now and doing the rest of the pruning?